modin/core/io/io.py

```diff
@@ -126,17 +114,10 @@ def from_dataframe(cls, df):
         summary="Load a parquet object from the file path, returning a query compiler",
         returns=_doc_returns_qc,
     )
-    def read_parquet(
-        cls, path, engine, columns, storage_options, use_nullable_dtypes, **kwargs
-    ):  # noqa: PR01
+    def _read_parquet(cls, **kwargs):  # noqa: PR01
```

Comment:
Why not just? 
```suggestion
    def read_parquet(cls, **kwargs):  # noqa: PR01
```

What are `Py36BaseIOCompat` and `LatestBaseIOCompat` for? The only difference I see in the interface (for example, `read_parquet`), why show it explicitly at this level? This difference in the interface is already at the level `modin/_compat/pandas/py36/io.py` and `modin/_compat/pandas/latest/io.py`.

Reply:
So we already have a layer where interfaces are mapped: `modin/_compat/pandas/py36/io.py` and `modin/_compat/pandas/latest/io.py`. What for to do mapping also in another level: `modin/_compat/io/py36/base_io.py` and `modin/_compat/io/latest/base_io.py`?

Reply:
Those are different I/O-s 🙃 `modin/_compat/_pandas` ones are for user-facing I/O interfaces while `modin/_compat/io` are for us _using_ pandas to do I/O for our implementations.